### PR TITLE
Remove unnecessary use of binascii

### PR DESCRIPTION
**Pull request by nsoranzo**
Also explicitly specify binary mode for file opening and header strings (for Python3). xref. #1715 
Fix import order.

### CODE REVIEW

**Comment by jmchilton** [comment]
Very nice, :+1: from me!

**Comment by nsoranzo** [comment]
@galaxybot test this

File: lib/galaxy/datatypes/binary.py [hunk 1]
```diff
@@ -523,9 +522,9 @@ def set_meta( self, dataset, overwrite=True, **kwd ):
 
     def get_cram_version( self, filename):
         try:
-            with open( filename, "r") as fh:
+            with open( filename, "rb") as fh:
                 header = fh.read(6)
-                return ord( header[4] ), ord( header[5] )
+            return ord( header[4] ), ord( header[5] )
```
Comment (martenson) [inline lib/galaxy/datatypes/binary.py:527]: was this de-indented intentionally?

Comment (nsoranzo) [reply lib/galaxy/datatypes/binary.py:527]: Yes, because `fh` can be closed before returning.

**Comment by martenson** [comment]
thanks Nicola!

**Pull request merged**